docs/app/Examples/collections/Menu/Types/MenuExampleVerticalDropdown.js

```diff
@@ -13,7 +13,7 @@ export default class MenuExampleVerticalDropdown extends Component {
       <Menu secondary vertical>
         <Menu.Item name='account' active={activeItem === 'account'} onClick={this.handleItemClick} />
         <Menu.Item name='settings' active={activeItem === 'settings'} onClick={this.handleItemClick} />
-        <Dropdown as={Menu.Item} text='Display Options'>
+        <Dropdown text='Display Options' pointing='left' className='link item'>
```

Comment:
Here, we would want to use a prop `linkItem` rather than a `className='link item'`.  The Dropdown actually already has this prop in its source, it is just commented out.  We need to enable that prop and add a description to it.

Then, there is a test stubbed for this prop in `Dropdown-test.js`.  That also need to be uncommented and updated to pass.

The final result will be:

```jsx
<Dropdown text='Display Options' pointing='left' linkItem>
```